gn-tools-pipelines/templates/gntoolspipelines-job.yaml

```
@@ -15,11 +15,15 @@ spec:
         - name: {{ include "gn-tools-pipelines.fullname" . }}
           image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
           imagePullPolicy: {{ .Values.image.pullPolicy }}
+          {{- if .Values.tolerations }}
+          tolerations:
+            {{- .Values.tolerations | toYaml | nindent 8 }}
+          {{- end }}
```

can be simplified to:
```suggestion
          {{- with .Values.tolerations }}
          tolerations:
            {{- toYaml . | nindent 8 }}
          {{- end }}
```